README.md

```diff
@@ -9,24 +9,42 @@ it needs to be included while running them.
 
 1. Set the home environment variable
 
+    Mac:
+
     ```
     $ export EMISSION_SERVER_HOME=<path_to_emission_server_repo>
     ```
+    
+    In windows command prompt use:
+    
+    ```
+    > set EMISSION_SERVER_HOME=<path_to_emission_server_repo>
+    ``` 
```

Comment:
Do we have to support windows? We are not testing the windows codepaths regularly, and I am concerned that this will just bitrot and then we have to debug it all over again. Can we at least indicate that windows is unsupported and people are using it at their own risk?

Reply:
@allenmichael099 are you planning to resolve this so that I can merge it?